b2/raw_simulator.py

```diff
@@ -554,12 +554,12 @@ def get_download_authorization(
         self._assert_account_auth(api_url, account_auth_token, bucket.account_id)
         return {
             'bucketId':
-            bucket_id,
+                bucket_id,
             'fileNamePrefix':
-            file_name_prefix,
+                file_name_prefix,
             'authorizationToken':
-            'fake_download_auth_token_%s_%s_%d' %
-            (bucket_id, b2_url_encode(file_name_prefix), valid_duration_in_seconds)
+                'fake_download_auth_token_%s_%s_%d' %
+                (bucket_id, b2_url_encode(file_name_prefix), valid_duration_in_seconds)
```

Comment:
this gets poorly formatted due to missing , at the end of last value of multiline dict, please correct it